changelog.d/9905.misc

```diff
@@ -0,0 +1 @@
+Don't set the external cache for "joined hosts" if its been done recently.
```

Comment:
Can we say "Improve performance of [operations] for worker-based deployments using Redis", or something?